master/buildbot/test/util/scheduler.py

```diff
@@ -224,7 +224,8 @@ def sourceStampKey(sourceStamp):
         return self._addBuildsetReturnValue(builderNames)
 
     def fake_addBuildsetForChanges(self, waited_for=False, reason='', external_idstring=None,
-                                   changeids=None, builderNames=None, properties=None, **kw):
+                                   changeids=None, builderNames=None, properties=None,
+                                   priority=None, **kw):
```

Comment:
The purpose of this function is to register how `addBuildsetForChanges` would have been called if it was real. Thus add `priority` to addBuildsetForChanges list of dicts below.